lib/storage/src/content_manager/consensus_state.rs

```diff
@@ -173,17 +171,55 @@ impl<C: CollectionContainer> ConsensusState<C> {
 
         let mut stop_consensus: bool = false;
         for single_change in &change.changes {
-            if single_change.change_type == ConfChangeType::RemoveNode as i32 {
-                log::debug!("Remove node {}", single_change.node_id);
-                if self.this_peer_id() == single_change.node_id {
-                    stop_consensus = true;
+            match single_change.change_type() {
+                ConfChangeType::AddNode => {
+                    debug_assert!(
+                        self.peer_address_by_id()
+                            .get(&single_change.node_id)
+                            .is_some(),
+                        "Peer should be already known"
```

Comment:
is that because we always use `AddLearnerNode` first?

Reply:
Yes